src/platform/user/profile/vet360/components/VAPEmail.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
This file is based on the existing `Email` component, but configures and renders a `VAPEmailField` instead of a legacy `Vet360EmailField`